solidity/test/fixtures/bridge.ts

```diff
@@ -19,6 +21,10 @@ export default async function bridgeFixture() {
     await helpers.signers.getNamedSigners()
   const [thirdParty] = await helpers.signers.getUnnamedSigners()
 
+  const tbtc: TBTC = await helpers.contracts.getContract("TBTC")
+
+  const tbtcVault: TBTCVault = await helpers.contracts.getContract("TBTCVault")
```

Comment:
Shouldn't we also give ownership of `TBTC` to the `TBTCVault`? It would be good to mimic the real-world deployment as much as possible. Also, if we do that, we'll be able to skip this step in `TBTCVault.Redemption.test.ts`.

Reply:
Got it! Thanks for the clarification!